en/guide/vtol.md

```diff
@@ -0,0 +1,39 @@
+# VTOL Support
+
+DroneCore has basic support for VTOL vehicles. 
+
+> **Note** Most design and test effort has gone into multicopter support. 
+> [Get in touch](../README.md#getting-help) if you would like to help enhance the VTOL/Fixed-wing experience! 
+
+## Supported Functionality
+
+Much of the functionality in DroneCore is generic, and useful on all vehicle types. 
+At time of writing the only *VTOL-specific* functionality is the ability to 
+[transition to fixed wing mode (and back)](../guide/taking_off_landing.html#transition_vtol).
+
+This allows users to:
+
+- Take off and land in *multicopter* mode
+- Fly in multicopter mode or transition and fly in fixed wing mode. 
+- Fly missions in the currently mode when the mission is started.
+- Use the camera/gimbal and other generic features.
+
+
+## Remaining Work
+
+Known omissions:
+- Takeoff and landing is not supported in missions.

```

Comment:
Takeoff is fine for multicopter mode. You set a mission and the takeoff waypoint is set automatically.

Reply:
1. So how does that sort of takeoff differ from [MAV_CMD_NAV_VTOL_TAKEOFF](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_VTOL_TAKEOFF)
2. After doing the MC takeoff can you send the transition command and proceed as VTOL?